onnxruntime/test/custom_op_registration/test_registercustomops.cc

```diff
@@ -40,6 +43,26 @@ TEST(CustomOpRegistration, TestUsingFuncNameCApi) {
   Ort::Session session(*ort_env, TestModel, session_options);
 }
 
+#if !defined(DISABLE_FLOAT8_TYPES)
+
+TEST(CustomOpRegistration, TestUsingFuncNameCApiFloat8) {
+  Ort::SessionOptions session_options;
+
+  // need to reference something in the custom ops library to prevent it being thrown away by the linker
+  void* addr = reinterpret_cast<void*>(RegisterCustomOpsAltName);
+  std::cout << "RegisterCustomOpsAltName addr " << addr << "\n";
+
+  // RegisterUnitTestCustomOps will add the Foo custom op in domain ort_unit_test
+  // custom_op_library has RegisterCustomOps and RegisterCustomOpsAltName as options for registration functions.
+  // Call RegisterCustomOpsAltName to test the path which does not use the default name.
+  Ort::ThrowOnError(Ort::GetApi().RegisterCustomOpsUsingFunction(session_options, "RegisterCustomOpsAltName"));
```

Comment:
>rt::GetApi().RegisterCustomOpsUsingFunctio [](http://example.com/codeflow?start=21&length=42)

There should be a C++ interface?

Reply:
I'm not sure I understood what you meant. In the unit tests, I added a model using Float8E4M3FN as inputs and outputs and made sure it is working. Did I miss anything?

Reply:
You are invoking a C interface directly. This code serves as an example for others. Can you casll session_options.RegisterCustomOpsUsingFunction() CXX interface which would be more readable anyway.

Reply:
Ok. What about the test just before this one? It is the same code except the model name which is different.

Reply:
I think we should not use C interfaces in C++ code.